src/test/ui/consts/std/alloc.32bit.stderr

```diff
@@ -34,7 +34,7 @@ LL | const LAYOUT_SIZE_NEGATIVE_ONE: Layout = unsafe { Layout::from_size_align_u
 error[E0080]: it is undefined behavior to use this value
   --> $DIR/alloc.rs:21:1
    |
-LL | const LAYOUT_SIZE_HIGH_BIT: Layout = unsafe { Layout::from_size_align_unchecked((isize::MAX as usize) + 1, 1) };
+LL | const LAYOUT_SIZE_HIGH_BIT: Layout = unsafe { Layout::Layout::from_size_align_unchecked(SIZE_MAX + 1, 1) };
```

Comment:
What happened here?

Reply:
Oops, bad copy-paste.  Thanks for catching!